### ISSUE TITLE
[MCU] LED driver

### ISSUE THREAD

**Issue by AndreasFreund** [amosproj/amos2024ss07-updating-flash-boot-loader#9]
## User story
1. As a user
2. I want to see the state and activity of the bootloader via LEDs or similar
3. So that I can be sure that everything is working correctly or diagnose issues

## Acceptance criteria
* .h-file without platform specific includes defines functions for controlling leds, e.g. led_activity_toggle(), ...
* .h-file for platform has initialization method
* .c-file implements initialization (except initializing GPIO peripheral, which must be done before) and functionality

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
